shared/oae/css/oae.base.css

```diff
@@ -904,24 +904,27 @@ ul.as-list li.as-result-item.active {
 
 .oae-markdown-embedded table {
     border-bottom: 1px solid #ECECEC;
-    font-size: 14px;
     margin: 20px 0;
 }
 
 .oae-markdown-embedded table th {
     background-color: #FFF;
     border-bottom: 1px solid #999;
     border-top: 1px solid #ECECEC;
-    font-weight: bolder;
+    font-weight: 500;
```

Comment:
Yeah, I noticed, but it needs to visually match the design. I've changed it to 600 and will be merging this.